docs/source/reference/io.rst

```diff
@@ -17,6 +17,16 @@ NumPy binary files (NPY, NPZ)
    savez
    savez_compressed
 
+Text files
+-----------------------------
+
+.. autosummary::
+   :toctree: generated/
+
+   loadtxt
+   savetxt
+   genfromtxt
+   fromstring
 
 String formatting
```

Comment:
`array2string` needs to be added to this section.
https://numpy.org/doc/stable/reference/routines.io.html